README.md

```diff
@@ -1,24 +1,59 @@
-# README
+# GlitchCraft
+GlitchCraft is a Ruby on Rails application allowing users to glitch images. Users can search for images based off a keyword that will return two images for the user's choosing. The first image is created by an AI from scratch based on the keyword via [DeepAI](https://deepai.org/machine-learning-model/text2img), while the second is returned based off search results from human generated image tags in [pixabay](https://pixabay.com/en/photos/search/). This gives user's the option to glitch an image that resembles human perceived notions or an AI's generated notions. GlitchCraft uses a combination of JavaScript and Canvas to glitch the images and employs Google OAuth to authenticate users. Images are stored via ActiveStorage and can be downloaded to the user's computer.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Relevant links
+[View our app!](https://glitch-craft.herokuapp.com/)
 
-Things you may want to cover:
+[GlitchService Microservice](https://damp-forest-93176.herokuapp.com/)
 
-* Ruby version
+[GlitchService GitHub](https://github.com/Kathybui732/glitch-service)
 
-* System dependencies
+## Versions
+Ruby 2.5.3
+Rails 5.2.4.4
 
-* Configuration
+## Configuration
+1. Run `git clone git@github.com:sciencefixion/glitch_craft.git`
+2. `cd glitch_craft`
+3. `bundle install`
+4. `rake db:{create,migrate}`
+5. `bundle exec figaro install`
+6. Add your google client secret and ID to the application.yml file
+7. `rails s`
 
-* Database creation
+## Test Suite
+1. Run `bundle exec rspec`
 
-* Database initialization
+## Microservices
+This app depends on the [GlitchService microservice](https://damp-forest-93176.herokuapp.com/). GlitchService works in hand with GlitchCraft to consolidate two images that users can choose from to glitch.
 
-* How to run the test suite
+## Contributors
 
-* Services (job queues, cache servers, search engines, etc.)
+[Dan Sehl's GitHub](https://github.com/dtsehl) &
+[Linkedin](https://www.linkedin.com/in/danielsehl)
 
-* Deployment instructions
+[Jane Greene's Github](https://github.com/janegreene) &
+[Linkedin](https://www.linkedin.com/in/jane-greene-mba/)
 
-* ...
+[Kathy Bui's Github](https://github.com/Kathybui732) &
+[Linkedin](https://www.linkedin.com/in/kathy-bui-87a27a1ab/)
+
+[Dorion's GitHub]() &
+[Linkedin]()
+
+## Our App
+
+### Schema
```

Comment:
https://dbdiagram.io/home we can use this once we get the users and images related.